nodeup/pkg/model/kube_proxy.go

```diff
@@ -129,6 +129,21 @@ func (b *KubeProxyBuilder) buildPod() (*v1.Pod, error) {
 		return nil, fmt.Errorf("Error parsing CPURequest=%q", c.CPURequest)
 	}
 
+	cpuLimit, err := resource.ParseQuantity(c.CPULimit)
```

Comment:
@horaceheaven we shouldn't call ParseQuantity if these values are empty strings.